symfony-reproducer.md

```diff
@@ -0,0 +1,111 @@
+# Creating a Symfony Reproducer
+
+Oh snap! You've discovered what you suspect is a bug in Symfony. Don't despair, Symfony is open source so
+there many ways you can help get it resolved. The following is a general list of steps you can take:
+
+1. Search the [symfony/symfony](https://github.com/symfony/symfony) repository for keywords related to your problem.
+   If you're seeing an unexpected exception, this message could be a good thing to search. It's possible this bug has
+   already been reported and maybe a fix is already in progress. Be sure to search issues, pull requests and discussions.
+2. Ask about your issue in the `#support` channel on the [Symfony Slack](https://symfony.com/slack). This can be a good
+   way to confirm the bug or perhaps get ideas for a workaround. Keep in mind that users who respond in this channel are
+   volunteers. Don't be discouraged if you don't get an answer.
+3. Do you think you know how to fix the issue? Open a PR! See the
+   [Symfony documentation](https://symfony.com/doc/current/contributing/code/pull_requests.html) to help with this.
+   Be sure to add a test if possible as this will likely be asked of you during the review.
+4. Not sure how to fix? Are you able to create a failing test? Same as the above step, create a PR that demonstrates
+   the bug in a failing test. Even if it doesn't include a fix, it's highly likely it will get reviewed and confirmed.
+   You'll also likely get tips on how to fix!
+5. Open a [bug report issue](https://github.com/symfony/symfony/issues/new/choose) and clearly explain the issue
+   you are facing. Initially including a reproducer (as we'll build below) is a great way to make your issue stand out!
+
+> [!TIP]
```

Comment:
I believe these need to be converted to the SymfonyCasts abomination syntax before publishing.